components/tools/OmeroPy/test/integration/gatewaytest/test_get_objects.py

```diff
@@ -571,6 +571,33 @@ def testGetImage(self, gatewaywrapper, author_testimg_tiny):
         assert image.getOwnerOmeName == testImage.getOwnerOmeName
         assert image.getThumbVersion() is not None
 
+    @pytest.mark.parametrize("load_pixels", [True, False])
+    @pytest.mark.parametrize("load_channels", [True, False])
+    def testGetImageLoadPixels(self, load_pixels, load_channels,
+                               gatewaywrapper, author_testimg_tiny):
```

Comment:
so we should probably find a better solution since we are trying to get of rid of tinytest